ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -182,3 +182,28 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None, axis
 @to_ivy_arrays_and_back
 def single(x):
     return ivy.astype(x, ivy.float32)
+
+
+@to_ivy_arrays_and_back
+def array_str(a, max_line_width=None, precision=None, suppress_small=False):
+    if precision is None:
+        # TODO: need implementation of get_printoptions() functional API
+        precision = ivy.get_printoptions()['precision']
+    def round_suppress(ele, precision_):
+        count_after_decimal = str(ele)[::-1].find('.')
+        if suppress_small and (count_after_decimal > precision_):
+            # supress to zero
+            return 0
+        else:
+            return round(ele, precision_)
+
+    a = list(map(round_suppress, a, precision))
+    a = str(a)
+    
+    if max_line_width is None:
+        # TODO: need implementation of get_printoptions() functional API
+        max_line_width = ivy.get_printoptions()['linewidth']
```

Comment:
we don't have a function for this yet but you can hardcode it as `75` for the time being!